services/audit-service/src/main/java/datawave/microservice/audit/replay/ReplayController.java

```diff
@@ -0,0 +1,726 @@
+     *
+     * @param id
+     *            The audit replay id
+     * @return status, indicating whether the audit replay was successfully resumed
+     */
+    @ApiOperation(value = "Resumes an audit replay.")
+    @RolesAllowed({"Administrator", "JBossAdministrator"})
+    @RequestMapping(path = "/{id}/resume", method = org.springframework.web.bind.annotation.RequestMethod.POST)
+    public String resume(@ApiParam("The audit replay id") @PathVariable("id") String id, HttpServletResponse response) {
+        
+        log.info("Resuming audit replay with id " + id);
+        
+        String resp;
+        Status status = status(id, replayProperties.isPublishEvents());
+        if (status != null) {
+            if (resume(status)) {
+                resp = "Resumed audit replay with id " + id;
+            } else {
+                response.setStatus(HttpServletResponse.SC_BAD_REQUEST);
+                resp = "Cannot resume audit replay with id " + id;
+            }
+        } else {
+            response.setStatus(HttpServletResponse.SC_BAD_REQUEST);
+            resp = "No audit replay found with id " + id;
+        }
+        
+        log.info(resp);
+        return resp;
+    }
+    
+    private boolean resume(Status status) {
+        // if the audit replay is idle or stopped, start it
+        if (status.getState() == IDLE || status.getState() == STOPPED)
+            runningReplays.put(status.getId(), start(status));
+        else
+            return false;
+        
+        return true;
+    }
+    
+    /**
+     * Resumes all audit replays
+     *
+     * @return status, indicating the number of audit replays which were successfully resumed
+     */
+    @ApiOperation(value = "Resumes all audit replays.")
+    @RolesAllowed({"Administrator", "JBossAdministrator"})
+    @RequestMapping(path = "/resumeAll", method = org.springframework.web.bind.annotation.RequestMethod.POST)
+    public String resumeAll() {
+        
+        log.info("Resuming all audit replays");
+        
+        int replaysResumed = 0;
+        
+        // resume all of our replays
+        List<Status> statuses = statusAll().stream().filter(status -> status.getState() == IDLE || status.getState() == STOPPED).collect(Collectors.toList());
+        for (Status status : statuses) {
+            runningReplays.put(status.getId(), start(status));
+            replaysResumed++;
+        }
+        
+        String resp = replaysResumed + " audit replays resumed";
+        log.info(resp);
+        return resp;
+    }
+    
+    /**
+     * Deletes an audit replay
+     *
+     * @param id
+     *            The audit replay id
+     * @return status, indicating whether the audit replay was successfully deleted
+     */
+    @ApiOperation(value = "Deletes an audit replay.")
+    @RolesAllowed({"Administrator", "JBossAdministrator"})
+    @RequestMapping(path = "/{id}/delete", method = org.springframework.web.bind.annotation.RequestMethod.POST)
```

Comment:
Usually, delete methods would use the DELETE verb. Is there a reason you aren't?

Reply:
Updated in the next commit.